sky/skylet/job_lib.py

```diff
@@ -291,7 +291,7 @@ def get_latest_job_id() -> Optional[int]:
 
 
 def get_job_time_payload(job_id: int, is_end: bool) -> Optional[int]:
-    field = 'end_at' if is_end else 'start_at'
+    field = 'end_at' if is_end else 'submitted_at'
```

Comment:
Should we be concerned about this change?

Many places encode the return value as start_at, which has downstream usage/assumptions:
- https://github.com/skypilot-org/skypilot/blob/8a80f5be49d951d1e6334f2cd1e6595f65e4e413/sky/spot/controller.py#L67
- etc

Reply:
Also, confused why we need to do this change. Let's say we submit 1000 spot jobs, thus making many of them pending. Would this change somehow start counting those pending jobs' duration?

Reply:
> Should we be concerned about this change?
> Many places encode the return value as start_at, which has downstream usage/assumptions:
> https://github.com/skypilot-org/skypilot/blob/8a80f5be49d951d1e6334f2cd1e6595f65e4e413/sky/spot/controller.py#L67

Thanks for catching this! I changed the variable name to `launched_at` instead, but still keep the concept for spot job to be `start_time`, as we now consider the setup to happen after the the spot job starts.

> Also, confused why we need to do this change. Let's say we submit 1000 spot jobs, thus making many of them pending. Would this change somehow start counting those pending jobs' duration?

The pending period for the spot job does not count for the jobs' duration, because:
1. Each spot cluster will be dedicated for a single spot job, which means the job will not be in JobStatus.PENDING mode when the spot cluster is healthy.
2. 1000 spot jobs will be pending on the controller, i.e. in the job queue of the controller. The submitted_at/start_at are only retrieved from the spot cluster, after the spot job is actually submitted to a spot cluster. The only change to the job duration is from the duration for the `run` section to the duration of all `setup` and `run` section.